backend/open_webui/functions.py

```
@@ -28,7 +28,10 @@
 from open_webui.models.functions import Functions
 from open_webui.models.models import Models
 
-from open_webui.utils.plugin import load_function_module_by_id
+from open_webui.utils.plugin import (
+    load_function_module_by_id,
```

Ruff (0.12.2)

`open_webui.utils.plugin.load_function_module_by_id` imported but unused

Remove unused import: `open_webui.utils.plugin.load_function_module_by_id`

(F401)